src/azul/service/responseobjects/hca_response_v5.py

```diff
@@ -626,9 +620,13 @@ def __init__(self, raw_response):
             donorCount=self.agg_contents(self.aggregates, 'donorCount', agg_form='value'),
             labCount=self.agg_contents(self.aggregates, 'labCount', agg_form='value'),
             totalCellCount=self.agg_contents(self.aggregates, 'total_cell_count', agg_form='value'),
-            organTypes=list(map(OrganType.for_bucket, _organ_types['buckets'])),
+            organTypes=list(map(OrganType.for_bucket,
+                                _organ_types['buckets'],
+                                repeat(tuple(_organ_types['meta']['path'])))),
```

Comment:
This assumes that certain aggregation have the `meta.path` entry. That is still too brittle. The rule of thumb is that if you find yourself adding code that mentions concrete document fields or their path, you're doing it wrong. :-)